package.json

```diff
@@ -7,12 +7,10 @@
   "scripts": {
     "build": "babel src -d dist",
     "prepack": "npm run build",
-    "test": "npm run test:functional:start && npm run test:functional && npm run test:functional:stop",
+    "test": "npm run test:functional",
     "test:functional": "node ./test/functional/environment.js",
-    "test:functional:start": "node ./test/functional/environment.js --start-environment",
-    "test:functional:stop": "node ./test/functional/environment.js --stop-environment",
```

Comment:
Maybe we should still keep the commands to manually start and stop testing environment? 🤔

Reply:
Just little worried about https://github.com/pipedrive/api-docs/blob/master/scripts/sdk-validator.js#L119-L139 .Did you try to run the tests in api-docs also?

Reply:
I didn't run it in api-docs. but if we leave the start and stop commands wouldn't it create 3 instances of tests ?, since the main file runs the test in series by starting the env,  running the test and stopping the env

Reply:
In api-docs we also trigger the client-nodejs tests. You can follow the README there to try it.

Reply:
Don't really know about the instances. I know that tests are run with 3 different node version and yes I was told that they run parallel https://pipedrive.slack.com/archives/CHCMSJ4NN/p1638791959006000

Reply:
And I was told that using random port should fix the problem

Reply:
i tried first just doing the random port change but it was failing for me so that's why i had to change the logic to first start the mockServer then run the tests. but i can look into it again